src/sql/base/browser/ui/table/formatters.ts

```diff
@@ -170,6 +143,55 @@ export function slickGridDataItemColumnValueWithNoData(value: any, columnDef: an
 	};
 }
 
+export function treeGridExpandableColumnFormatter<T>(formattingFunction: Slick.Formatter<T>) {
```

Comment:
1. Add a doc comment explaining what this one is for and why it's different than the others
2. Would suggest `createTreeGridExpandableColumnFormatter` as the name - this function isn't the formatter itself
3. Add return type